CGATPipelines/PipelinePreprocess.py

```diff
@@ -512,23 +516,50 @@ def build(self, infiles, outfiles, output_prefix):
         assert len(infiles) == len(outfiles)
 
         cmds = []
-        if int(untrimmed) == 0:
-            for infile, outfile in zip(infiles, outfiles):
+        if self.process_paired and len(infiles) == 2:
+            in1, in2 = infiles
+            out1, out2 = outfiles
+
+            if "fastq" in in1:
+                format = "--format=fastq"
+            elif "fasta" in in1:
+                format = "--format=fasta"
+            else:
+                format = ""
+
+            untrimmed_output1, untrimmed_output2 = \
+                [i.replace(".fast", "_untrimmed.fast")
+                 for i in infiles]
+
+            if untrimmed:
+                processing_options += \
+                    "--untrimmed-output=%(untrimmed_output1)s" \
+                    "--untrimmed-paired=%(untrimmed_output2)s" % locals()
```

Comment:
Should line 537 be `"--untrimmed-output-paired=%(untrimmed_output2)s" % locals()`?

`cutadapt --help|grep untrimmed`
    --no-trim           Match and redirect reads to output/untrimmed-output as
    --discard-untrimmed, --trimmed-only
    --untrimmed-output=FILE
    --untrimmed-paired-output=FILE
                        together with --untrimmed-output when trimming paired-


`cutadapt --version`
1.9.1